notification/slack.py

```diff
@@ -24,7 +25,7 @@
 description:
     - The M(slack) module sends notifications to U(http://slack.com) via the Incoming WebHook integration
 version_added: 1.6
-author: "Ramon de la Fuente (@ramondelafuente)"
+author: Ramon de la Fuente <ramon@delafuente.nl>, Stefan Berggren <nsg@nsg.cc>
```

Comment:
also, author should be a list when more than one:
author: 
   - Ramon de la Fuente <ramon@delafuente.nl>
   - Stefan Berggren <nsg@nsg.cc>